<center>
    <img src="https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBMDeveloperSkillsNetwork-DS0105EN-SkillsNetwork/labs/Module2/images/SN_web_lightmode.png" width="300" alt="cognitiveclass.ai logo"  />
</center>


#### Add your code below following the instructions given in the course


# My Jupyter Notebook on IBM Watson Studio 

**Friedman COUTHON**

Ingineer in Applied Mathmatics, Data Scientist in training

_I want to be a Data Scientist because using data to explain and solve real world problems is something I really enjoy._

### A function to calculate the factorial of a number using recursion.

In [3]:
def fact(n):
    if n == 1:
        return n
    else:
        return n*fact(n-1)


num = 5

# check if the given number is negative
if num < 0:
    print("Factorial for negative number does not exist !")
elif num == 0:
    print("The factorial of 0 is 1")
else:
    print("The factorial of", num, "is : ", fact(num))

The factorial of 5 is :  120


* 1x2=2
* 2x3=6
* 6x4=24
* 24x5=120

| Days               | Work        | Rest          |
| :---               |    :----:   |          ---: |
| Monday to Friday   | Yes         | No            | 
| Saturday and Sunday| No          | Yes           |

~~The table above is useful.~~